src/cmd/ksh93/meson.build

```diff
@@ -95,7 +95,7 @@ all_tests = [
     ['basic', 90], ['bracket'], ['builtins'], ['case'], ['comvar'],
     ['comvario'], ['coprocess', 50], ['cubetype'], ['directoryfd'], ['enum'],
     ['exit'], ['expand'], ['functions'], ['glob'], ['grep'], ['heredoc'],
-    ['io'], ['locale'], ['math', 50], ['nameref'], ['namespace'],
+    ['io'], ['locale'], ['math', 50], ['nameref'], ['namespace'], [ 'modifiers' ],
```

Comment:
Small style nit: Spacing between the square brackets when everywhere else there is no spacing.